data/companies.json

```diff
@@ -592,6 +592,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "NepalLink Network",
+      "establishedAt": 1106467200,
+      "location": "Kathmandu, Nepal",
+      "website": "https://www.nepallink.net/",
+      "email": "info@nepallink.net",
+      "phone": "+977-14260822",
+      "socials": {
+      "facebook": "https://www.facebook.com/ncompute/",
```

Comment:
To make this section's formatting better, use prettier.

Reply:
Sure I will fix this section @susanhsrestha